Lib/test/test_zipfile/test_path.py

```diff
@@ -161,10 +150,10 @@ def test_open_encoding_utf16(self):
         u16 = path.joinpath("16.txt")
         with u16.open('r', "utf-16") as strm:
             data = strm.read()
-        self.assertEqual(data, "This was utf-16")
+        assert data == "This was utf-16"
```

Comment:
I acknowledge these bare asserts are an anti-pattern for CPython generally. I plan to publish a treatise on these in my blog, but for now I'm continuing to use them in the backports I maintain.